rust/gitxetcore/src/config/cas.rs

```diff
@@ -54,23 +55,37 @@ impl TryFrom<Option<&Cas>> for CasSettings {
                     }
                     None => None,
                 };
-                (endpoint, prefix)
+
+                let size_threshold = match x.sizethreshold {
```

Comment:
low prio, more rusty way to do this

Reply:
Ok, I've replaced the match with an `Option.map(...).transpose()?` chain.